db/migrations/0002_alter_player_guild.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2024-12-03 05:05
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='players', to='db.guild'),
```

Comment:
The foreign key reference `to='db.guild'` is incorrect. It should reference the correct app name containing the `Guild` model. For example, if your app is named `myapp`, it should be `to='myapp.Guild'`.